src/routes/forms.js

```diff
@@ -173,6 +174,35 @@ export default [
         payload: createLiveSchema
       }
     }
+  },
+  {
+    method: 'POST',
+    path: '/forms/{id}/create-draft',
```

Comment:
Up for discussion perhaps?

Always thought these little updates were PATCH

Perhaps with a minimal payload `{ author, status: 'live' }`

```suggestion
    method: 'PATCH',
    path: '/forms/{id}',
```

Reply:
Arguably yes. The upcoming `PATCH /forms/{id}` to updated the `title`, `teamName` etc. will be certainly be a `PATCH` because this is patching an existing resource (the db record).

Here we are updating a resource (the draft/live parts of the metadata) but we're also creating a new resource (the S3 file). While that's an internal implementation detail, I think from the outside consumers point of view, we are creating a new resource hence POST.